### ISSUE TITLE
wider face annotations format (x1,y1, width, height)?

### ISSUE THREAD

**Issue by hkdqliu** [Seanlinx/mtcnn#32]
Hi,@Seanlinx, I'm new to your mtcnn-master. In your get_pnet_data.py , the format of wider face annotations is (x1,y1,x2,y2), but one of the contribitor of the WIDER FACE dataset , yangshuo, he told me the format of wider face annotations is (x1,y1, width, height)
![11](https://cloud.githubusercontent.com/assets/27625500/25071390/34110b48-22e8-11e7-982d-89b3d7d793b9.JPG)

**Comment by Seanlinx** [comment]
@hkdqliu That's right, but I had it transformed. The scripts are located in prepare_data/wider_annotations

**Comment by hkdqliu** [comment]
OK,@Seanlinx, thanks!
